### PR TITLE
Add questionmark to display explanation

### PR DESCRIPTION
**Pull request by silkypriya**
**Description**
Adding a question mark next to "skip this image" button and Peer Review title which will display an explanation about their usage.

**Fixes** #2698 Improve peer review feature quality

**Tests performed**
Tested betaDebug on Realme 2 pro (Android Version 8.1.0) with API level 27.

**Screenshots of the result** 

Skip Button             |  Peer Review 
:-------------------------:|:-------------------------:
 |

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/review/ReviewActivity.java [hunk 1]
```diff
@@ -163,4 +174,25 @@ public void swipeToNext() {
...
+                ,getResources().getString(R.string.skip_this_image_explanation));
+    }
+
+    public void showReviewImageInfo(){
+        launchAlert(getResources().getString(R.string.review_image)
+                ,getResources().getString(R.string.review_image_explanation));
+    }
+
+    private void launchAlert(String title, String message){
+        new AlertDialog.Builder(ReviewActivity.this)
```
Comment (maskaravivek) [inline app/src/main/java/fr/free/nrw/commons/review/ReviewActivity.java]: Use `DialogUtil:showAlertDialog`

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -508,6 +508,10 @@ Upload your first media by tapping on the add button.</string>
     <string name="review_copyright_no_button_text">SEEMS FINE</string>
     <string name="review_thanks_yes_button_text">YES, WHY NOT</string>
     <string name="review_thanks_no_button_text">NEXT IMAGE</string>
+    <string name="skip_this_image">Skip this image</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml]: This string is duplicated

**Comment by silkypriya** [comment]
@maskaravivek @neslihanturan @domdomegg 
Commit following changes 
1. Used DialogUtil:showAlertDialog
2. Used info-icon instead of questionmark
3. Removed duplicate string 
Kindly review 
Thank you!

**Comment by silkypriya** [comment]
Hello @domdomegg,
Committed the required changes. Kindly review.
Thankyou!

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -508,6 +508,9 @@ Upload your first media by tapping on the add button.</string>
     <string name="review_copyright_no_button_text">SEEMS FINE</string>
     <string name="review_thanks_yes_button_text">YES, WHY NOT</string>
     <string name="review_thanks_no_button_text">NEXT IMAGE</string>
+    <string name="go_back_button_text">GO BACK</string>
```
Comment (domdomegg) [inline app/src/main/res/values/strings.xml]: Sorry - last change from me: this string is no longer needed.

**Comment by silkypriya** [comment]
Removed unused string.
Thankyou! @domdomegg

**Pull request merged**